code/modules/mob/living/living.dm

```diff
@@ -59,6 +59,15 @@
 		return 0
 	if(!..())
 		return 0
+	if(ishuman(src))
+		var/obj/item/hand_item = src.get_active_hand()
+		if(hand_item && istype(hand_item, /obj/item/weapon/gun) && A != hand_item)
+			if(src.a_intent == I_HELP || !ismob(A))
+				visible_message("<b>[src]</b> points to [A] with [hand_item]")
```

Comment:
Why did you even add this if you can only get to this point with guns? Just let them point normally.